### ISSUE TITLE
row-click issues and checkbox how to disable

### ISSUE THREAD

**Issue by dmotors** [matfish2/vue-tables-2#439]
- Vue.js version: 2.5.13
- consumed using: Laravel Mix

Main issues is:
1) I have a checkbox on first column and row-click will check it.
2) Only first row triggers row-click, subsequent rows does nothing.
3) If I was using row-click later I'd only want the checkbox column to be triggered not all columns.

Goal:
- I really just want to disable the row-click functionality altogether.

Here is a sample code:

	<v-server-table url="getinvdata" :columns="columns" :options="options" ref="myTable" @row-click="onRowClick($event)">
		<div slot="checkbox" slot-scope="props">
			<div>
				<input type="checkbox" :id="props.row.id" :value="props.row.id" v-model="checks">
			</div>
		</div>
	</v-server-table>	

So basically I do not need the row-click to even trigger at all I won't be using it in this case. However if I can also know why only first row is triggering row-click in case later I plan to use it but for now want to just disable it completely.

**Comment by matfish2** [comment]
I'm not sure I understood, so I'll focus on your stated goal:

> I really just want to disable the row-click functionality altogether.

Row click is not a function, it is an *event*. As such it has not functionality attached to it. It is up to you to decide how to handle the event. You can respond to it for certain rows and for others ignore it.
